backend/inspirehep/records/marshmallow/literature/latex.py

```diff
@@ -113,3 +122,29 @@ def get_dois(self, data):
         for doi_data in dois:
             doi_data["value"] = latex_encode(doi_data.get("value"))
         return dois or None
+
+    def get_book_publication_info(self, data):
+        if "book" in get_value(data, "document_type", []):
```

Comment:
better check `if not` - then you can instantly return and remove one indentation level. Also, no need for default get value `[]`

Reply:
At least when I test the latex_schema the ```[]``` default value is necessary.
```
if "book" not in get_value(data, "document_type"):
E       TypeError: argument of type 'NoneType' is not iterable
```

Reply:
yes, you are right, sorry